createRollupConfig.js

```diff
@@ -0,0 +1,121 @@
+import { terser } from 'rollup-plugin-terser';
+import { getBabelOutputPlugin } from '@rollup/plugin-babel';
+
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import json from '@rollup/plugin-json';
+import babel from '@rollup/plugin-babel';
+import peerDepsExternal from 'rollup-plugin-peer-deps-external';
+
+const { dependencies, peerDependencies } = require(__dirname + '/package.json');
+
+const external = [
```

Comment:
Tohle je presne to, co dela plugin `rollup-plugin-peer-deps-external`, takze se tehle logiky muzeme zbavit.